### PR TITLE
Update documentation for popcha

### PR DESCRIPTION
**Pull request by dayan9265**
This PR simply adds popcha in the structure in the README & remove an old comment

### CODE REVIEW

**Review by MeKHell — APPROVED** [review]
LGTM!

File: fe1-web/src/features/popcha/README.md
Comment (Tyratox) [inline fe1-web/src/features/popcha/README.md:1]: Typo at proove -> prove

sorry I can‘t highlight it since I am doing the review on my mobile phone 😂

**Review by Tyratox — APPROVED** [review]
LGTM except for the small typo ;)

File: fe1-web/src/features/popcha/README.md
Comment (dayan9265) [reply fe1-web/src/features/popcha/README.md:1]: Ah yes good catch, this spelling mistake should be fixed with the other PR on PoPCHA ^^

**Pull request merged**